### PR TITLE
Update to rocm smi lib

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

**Comment by jan-janssen** [comment]
@isuruf Can you take a look at this pull request?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,34 +1,48 @@
...
+    - test -f $PREFIX/lib/liboam$SHLIB_EXT.*.*  # [unix]
 
 about:
-  home: https://github.com/RadeonOpenCompute/ROC-smi
-  license: MIT
-  license_family: MIT
-  license_file: LICENSE
-  summary: 'ROC System Management Interface'
+  home: https://github.com/RadeonOpenCompute/rocm_smi_lib
+  summary: 'ROCm SMI LIB'
```
Comment (isuruf) [inline recipe/meta.yaml]: Can you keep the old summary. This is not only the library.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,34 +1,48 @@
...
+  description: |
+    The ROCm System Management Interface Library, or ROCm SMI library, is
+    part of the Radeon Open Compute ROCm software stack . It is a C library
+    for Linux that provides a user space interface for applications to 
+    monitor and control GPU applications.
```
Comment (isuruf) [inline recipe/meta.yaml]: Can you add that this has the `rocm-smi` executable as well?

File: recipe/build.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+mkdir -p build
+cd build
+cmake ${CMAKE_ARGS} ..
+make
```
Comment (isuruf) [inline recipe/build.sh]:
```suggestion
make -j${CPU_COUNT}
```

File: recipe/meta.yaml (hunk 1 above)
Comment (jan-janssen) [inline recipe/meta.yaml]:
```suggestion
  summary: 'ROC System Management Interface'
```

**Comment by jan-janssen** [comment]
@isuruf I updated the recipe based on your suggestions.

**Comment by isuruf** [comment]
Thanks

**Pull request merged**